apps/prairielearn/src/tests/questionSharing.test.ts

```diff
@@ -54,8 +54,8 @@ async function setSharingName(courseId: string, name: string) {
   });
 }
 
-async function accessSharedQuestionAssessment(course_id: string) {
-  const assessmentsUrl = `${baseUrl}/course_instance/${course_id}/instructor/instance_admin/assessments`;
+async function accessSharedQuestionAssessment(course_instance_id: string) {
+  const assessmentsUrl = `${baseUrl}/course_instance/${course_instance_id}/instructor/instance_admin/assessments`;
```

Comment:
This was an absolute nightmare to debug. 

For some reason, I had made the mistake of using course_instance_id's where I should have been using course Ids in a few places in this file. And for some reason it worked!

Now, with the addition of a second course instance to test course, it broke the fact that course instance ids' had, by dumb luck, been matching course ids.

Reply:
Oof, yeah, this is horrible. This would generally work because the example and test courses each only have a single course instance, and since they're synced in order, course 1 would get course instance 1 and course 2 would get course instance 2.

Reply:
Yup, that's right. 

It started out with "why the heck isn't auth working for a question preview? I didn't touch any of that code", and then a lot of debugging later I found this disaster.